packages/quick_actions/quick_actions_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: quick_actions_example
+description: Demonstrates how to use the quick_actions plugin.
+publish_to: none
+
+environment:
+  sdk: ">=2.15.0 <3.0.0"
+  flutter: ">=2.8.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  quick_actions_ios:
+    # When depending on this package from a real application you should use:
+    #   quick_actions: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
+
+dev_dependencies:
+  espresso: ^0.1.0+2
```

Comment:
This shouldn't be needed in the iOS version.

Reply:
Removed from iOS version.